src/client/method-status.c

```diff
@@ -280,7 +287,7 @@ static int method_status_unit_on(Client *client, char *node_name, char **units,
                 int r = get_status_unit_on(client, node_name, units[i], max_name_len);
                 if (r < 0) {
                         fprintf(stderr,
-                                "Failed to get status of unit %s on node %s - %s",
+                                "Failed to get status of unit %s on node %s - %s\n",
```

Comment:
What's the new line for? Wouldn't it be better to have the whole message on a single line?

Reply:
This one is not related to the issue.
But without this new line the output I get if the node is not connected to the controller is
`Failed to get status of unit virtnwfilterd.service on node fedora - Input/output erroradivak@fedora:~$`
so the new line will make it look like
`Failed to get status of unit virtnwfilterd.service on node fedora - Input/output error
adivak@fedora:~$`
I think it's look better and that why I added it.

Reply:
OK